### PR TITLE
Show name for bridged messages

### PR DESCRIPTION
**Pull request by gary-kim**
~~Requires https://github.com/42wim/matterbridge/pull/1506.~~ **Now merged!**

The way the current iteration works is that messages received from the `bridge-bot` user is labeled as Actor Type `bridged`. Actor Type `bridged` messages store the message author name in the Actor Id field in `oc_comments`. When serving to the frontend, this is slightly changed to put what is in Actor Id into Actor Display Name and set the Actor Id to `bridge-bot` (the reason it was done this way was to simplify the changes that would need to be made to clients).

**As far as clients have to care, the only difference is the addition of the Actor Type `bridged` and that for `bridged` messages, the Actor Display Name can be different for every message.**

### Screenshots
Screenshot 2021-05-28 at 09-52-29 Element  20  Test Room
Screenshot 2021-05-28 at 09-51-48 Test 2 - Talk - Nextcloud

### CODE REVIEW

File: lib/MatterbridgeManager.php
Comment (nickvergessen) [inline lib/MatterbridgeManager.php:386]: Does this mean all configs have to be recreated?

File: src/components/MessagesList/MessagesGroup/AuthorAvatar.vue
Comment (nickvergessen) [inline src/components/MessagesList/MessagesGroup/AuthorAvatar.vue:77]: Can you add this in `disableMenu` too, so the bridgebot has no avatar menu?

File: lib/MatterbridgeManager.php
Comment (gary-kim) [reply lib/MatterbridgeManager.php:386]: Yes. Until the config is recreated, it will continue to function as before.

**Comment by nickvergessen** [comment]
> Requires 42wim/matterbridge#1506

I guess we have to wait until that is merged?

**Comment by gary-kim** [comment]
Yup. It'd be noop if merged right now

**Comment by nickvergessen** [comment]
noop in terms of "works as before" or in terms of "breaks and doesn't do anything anymore"

**Comment by gary-kim** [comment]
Works as before

File: lib/Controller/ChatController.php
Comment (nickvergessen) [inline lib/Controller/ChatController.php:164]: Missing from documentation:
https://github.com/nextcloud/spreed/blob/master/docs/constants.md#actor-types-of-chat-messages

**Comment by gary-kim** [comment]
If everything seems fine on this side, we could merge now. It should function the same as before with a Matterbridge version that does not support the `SeperateDisplayName` config.

**Comment by nickvergessen** [comment]
Maybe you can ping them again?

**Comment by gary-kim** [comment]
https://github.com/42wim/matterbridge/pull/1506 is now merged so this feature should now work with the executables here: https://github.com/42wim/matterbridge/actions/runs/953056470.

**Review by nickvergessen — APPROVED** [review]
Good from my POV, but I guess we should wait for 22 to go out as the mobile apps need a patch too

**Comment by gary-kim** [comment]
Would have been nice to get it into 22 but we're lacking in time, I suppose

**Comment by nickvergessen** [comment]
Android patch at https://github.com/nextcloud/talk-android/pull/1355
iOS ticket raised in https://github.com/nextcloud/talk-ios/issues/579 but at least it shows something already.

**Comment by nickvergessen** [comment]
Can you rebase @gary-kim

**Pull request merged**
